recipes/neo4j-connector/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "neo4j-connector" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/textkernel/neo4j-connector/archive/v1.0.1.tar.gz
+  sha256: a89cb303762b7e82f4b54eea4a6b88e34cd88c8b88928517c2158a42041105a5
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
+    - pip
+  run:
+    - requests <3.0,>=2.0
```

Could you please add ``python`` as well?
```suggestion
    - requests <3.0,>=2.0
    - python
```